backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/ExpressionImpl.java

```diff
@@ -117,13 +119,16 @@ public Feature getResult() {
     /**
      * <!-- begin-user-doc --> <!-- end-user-doc -->
      *
-     * @generated
+     * @generated NOT
      */
     public Feature basicGetResult() {
-        // TODO: implement this method to return the 'Result' reference
-        // -> do not perform proxy resolution
-        // Ensure that you remove @generated or mark it @generated NOT
-        return null;
+            return this.getOwnedFeatureMembership().stream()
+                    .filter(ReturnParameterMembership.class::isInstance)
+                    .map(ReturnParameterMembership.class::cast)
+                    .map(ReturnParameterMembership::getOwnedMemberParameter)
+                    .filter(Objects::nonNull)
+                    .findFirst().orElseGet(this::getResultFromFeature);
```

Comment:
`.orElseGet(this::getResultFromFeature);` should be on a separated line

Reply:
Done